app/models/listed_taxon.rb

```diff
@@ -488,6 +490,41 @@ def trickle_down_establishment_means(options = {})
     ActiveRecord::Base.connection.execute(sql)
   end
   
+  def is_atlased?
+    if list.is_a?(CheckList) && list.is_default?
```

Comment:
So you're intentionally excluding listed taxa on non-default-but-comprehensive lists like http://www.inaturalist.org/check_lists/69089-Aizoaceae-of-Marin-County--CA--US, at least to start?